routes/index.js

```diff
@@ -127,6 +126,58 @@ module.exports = function (passport) {
     });
   });
 
+  router.post('/userManagement/deleteUser/:badge',function (req, res) {
+    if (!req.body) {
+      return res.sendStatus(400);
+    }
+    var DeletedBagde = req.params.badge;
+    dbAPI.deleteUser(DeletedBagde).then(function (result) {
+      console.log(result);
+      console.log("user is deleted successfully");
+  });
+    res.redirect('/userManagement');
+  });
+
+  router.post('/userManagement/modifyUser/:badge',function (req, res) {
```

Comment:
I think we should change this route to '/userManagement/confirmUser/:badge' instead of modifyUser